app/javascript/packs/shared/modals_container/modals/LeaveTeamModal.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import PropTypes, { bool, number, string, func } from "prop-types";
+import { Modal, Button, Alert, Glyphicon } from "react-bootstrap";
+import { FormattedMessage, FormattedHTMLMessage } from "react-intl";
+import { connect } from "react-redux";
+import axios from "../../../app/axios";
+
+import { LEAVE_TEAM_PATH } from "../../../app/routes";
+import { leaveTeamModalShow } from "../../actions/LeaveTeamActions";
+import { addTeamsData, setCurrentTeam } from "../../actions/TeamsActions";
+
+class LeaveTeamModal extends Component {
+  constructor(props) {
+    super(props);
+    this.onCloseModal = this.onCloseModal.bind(this);
+    this.leaveTeam = this.leaveTeam.bind(this);
+  }
+
+  onCloseModal() {
+    this.props.leaveTeamModalShow(false);
+  }
+
+  leaveTeam() {
+    const teamUrl = `${LEAVE_TEAM_PATH}?team=${this.props.teamId}`;
+    axios
+      .delete(teamUrl, {
+        withCredentials: true
+      })
+      .then(response => {
+        console.log(response);
+        const teams = response.data.teams.collection;
+        this.props.addTeamsData(teams);
+        const currentTeam = _.find(teams, team => team.current_team);
```

Comment:
I think you need braces around _.find((teams, team) => ...

only if you have one variable in fat arrow function you can leave braces off

Reply:
first arg is array the second is function

Reply:
:+1: